UNRELEASED_CHANGELOG.md

```diff
@@ -100,9 +101,14 @@
 - Added updated logic to Link preview attachments, which chooses either the `titleLink` or the `ogUrl` when loading the data, depending on which exists .
 
 ### ✅ Added
+- Added lots of improvements to Avatars - added a `UserAvatar`, `ChannelAvatar` and an `InitialsAvatar` to load different types of data.
+- We now show a matrix of user images in case we're in a group DM.
+- We also show initials in case the user doesn't have an image.
 
 ### ⚠️ Changed
 - `ViewModel`s now initialize automatically, so you no longer have to call `start()` on them. This is aimed to improve the consistency between our SDKs.
+- Added a `Shape` parameter to `Avatar` to customize the shape.
+- The `User` parameter in the `ChannelListHeader` is now non-nullable.
```

Comment:
The UI Components library allows hiding the avatar from the header as a customization option (with attributes). Perhaps the Compose SDK should have a slot for the content that shows up at the start of the header?

Reply:
We could expose a property for the start of the header. I think we can then move back to the nullable user and just show a spacer in case there's nothing (to make sure things are aligned)